apps/coscrad-frontend-e2e/src/e2e/resources/vocabulary-lists/vocabulary-list.ADD_TERM_TO_VOCABULARY_LIST.e2e.cy.ts

```diff
@@ -0,0 +1,90 @@
+import { AggregateType, LanguageCode, ResourceType } from '@coscrad/api-interfaces';
+import { buildDummyAggregateCompositeIdentifier } from '../../../support/utilities';
+
+const commandLabel = 'Add Term to Vocabulary List';
+
+const existingVocabularyListName = 'birds';
+
+const languageCodeForVocabularyListName = LanguageCode.English;
+
+const aggregateCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.vocabularyList,
+    1
+);
+
+const textForTermToAdd = 'hello universe';
+
+const languageCodeForTerm = LanguageCode.English;
+
+const termAggregateCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.term,
+    2
+);
+
+const buildDetailRoute = (id: string) => `/Resources/VocabularyLists/${id}`;
+
+describe(`ADD_TERM_TO_VOCABULARY_LIST`, () => {
```

Comment:
I believe there were issues with this test. We should either fix it or `.skip` for now. We've done manual testing, but the goal is to have good coverage of the UX portion of these flows in Cypress to prevent regressions in client behaviour.

Reply:
I've skipped it with a comment to this effect.